data_diff/diff_tables.py

```diff
@@ -85,6 +85,7 @@ class DiffStats:
     table2_count: int
     unchanged: int
     diff_percent: float
+    extra_column_diffs: Dict[str, int]
```

Comment:
```suggestion
    extra_column_diffs: Optional[Dict[str, int]]
```
Should fix the CI error